RELNOTES

```diff
@@ -8,6 +8,7 @@ firejail (0.9.58,2) baseline; urgency=low
   * new profiles: klavaro, vscodium
   * browser profiles fixes
   * various other bugfixes
+  * memory-deny-write-execute now also blocks memfd_create
```

Comment:
this needs to be moved into the 0.9.59 changelog above

Reply:
Thanks, fixed!